acra/guides/trying-acra-with-docker/_index.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Trying Acra with Docker
+bookCollapseSection: true
+---
+
+# Acra + Docker
+
+You can use containers with Acra. Acra works well with [Docker](https://www.docker.com/what-docker). However, we advise only going the "Acra in Docker containers" route for testing purposes, and NOT in a real-world production environment.
+
+This is why this document is called "Trying Acra with Docker", not "Using Acra with Docker in a Production Setting".
```

Comment:
This is also a fairly large document. I believe it will be more readable if split into multiple pages.